textile/chat-features.textile

```diff
@@ -104,121 +104,145 @@ The status of any given Chat Room is the combination of the states of all of its
+*** @(CHA-RL2k3)@ @[Testable]@ If the detachment is unsuccessful (i.e. @channel.detach()@ throws an error), the detach error will be thrown as the result of the room @detach()@ operation.
+** @(CHA-RL2e)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL2f)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL2g)@ This specification point has been removed. It was valid up until the single-channel migration.
+** @(CHA-RL2h)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL2h1)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL2h2)@ This specification point has been removed. It was valid up until the single-channel migration.
+*** @(CHA-RL2h3)@ This specification point has been removed. It was valid up until the single-channel migration.
 * @(CHA-RL3)@ A room must be explicitly released via the @RELEASE@ operation. This operation is not performed directly on a Room object by the client, but is described here.
+** @(CHA-RL3k)@ @[Testable]@ If a room lifecycle operation is already in progress, this operation shall wait until the current operation completes before continuing, subject to @CHA-RL7@.
 ** @(CHA-RL3a)@ @[Testable]@ If the room is already in the @RELEASED@ status, this operation is no-op.
 ** @(CHA-RL3b)@ @[Testable]@ If the room is in the @DETACHED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
 ** @(CHA-RL3j)@ @[Testable]@ If the room is in the @INITIALIZED@ status, then the room is immediately transitioned to @RELEASED@ and the operation succeeds.
 ** @(CHA-RL3c)@ This specification point has been removed.
 ** @(CHA-RL3i)@ This specification point has been removed.
-** @(CHA-RL3l)@ @[Testable]@ When the release operation commences, the room will transition into the @RELEASING@ status and any transient disconnect timeouts shall be cleared.
-** @(CHA-RL3d)@ @[Testable]@ All features channels must be detached in sequence.
-** @(CHA-RL3e)@ @[Testable]@ If a channel is in the @FAILED@ state when it is time to detach, it shall be ignored.
-** @(CHA-RL3f)@ @[Testable]@ If a channel fails to detach (i.e. the call to @detach()@ returns an error), the @CHA-RL3d@ channel detach sequence will be retried after a 250ms delay. Retries are continued until @CHA-RL3g@ is met.
-** @(CHA-RL3g)@ @[Testable]@ Once all channels have entered the @DETACHED@ (i.e. the call to @detach()@ succeeds) or @FAILED@ (i.e. the call to @detach()@ fails and the channel state is subsequently @FAILED@) state, then the room state is transitioned to @RELEASED@ and the operation completes.
+** @(CHA-RL3m)@ @[Testable]@ When the release operation commences, the room will transition into the @RELEASING@ status.
+** @(CHA-RL3n)@ @[Testable]@ After @CHA-RL3m@, the room enters a detach loop.
```

Comment:
Not sure this one needs to be `Testable`; seems more like an overview of the subpoints than a behaviour to be tested.

Reply:
Removed!